qemu/target-mips/op_helper.c

```diff
@@ -47,6 +47,10 @@ static inline void QEMU_NORETURN do_raise_exception_err(CPUMIPSState *env,
         cpu_restore_state(cs, pc);
     }
 
+    if(exception == 0x11){
```

Comment:
nice work, just a bit fix here about coding style: we need a space between `if` and `(`, and another space between `)` and `{`.